### PR TITLE
Chest X-ray - Tuberculosis Disease Prediction

### PR DESCRIPTION
**Pull request by aaradhyasinghgaur**
**Closes :-** #138
**What I've done :-**

  1. Data Preparation: Collect and preprocess a balanced dataset of real and fake face images, including normalization, resizing, and augmentation.
   2.  Base Model Selection: EfficientNetB0,VGG16 , Xception , InceptionV3 like 5 different models excluding its top layers, to leverage its learned features.
  3.  Model Construction: Add custom layers on top of the base model for binary classification, compiling with appropriate loss and metrics.
   4. Initial Training: Train the model with the base layers frozen to only update the new layers.
   5. Fine-Tuning: Unfreeze some or all of the base model layers and continue training with a lower learning rate to fine-tune the entire network.
    6.) EDA analysis.
    7.) Comaprioson using performance matrices such as accuracy scores , confusion matrix etc.

### CODE REVIEW

**Comment by aaradhyasinghgaur** [comment]
@TAHIR0110 kindly review my pr and merge it in case of no problems please.

**Comment by TAHIR0110** [comment]
merged!
